install_config/cluster_metrics.adoc

```diff
@@ -306,15 +340,15 @@ ifdef::openshift-enterprise[]
 ====
 endif::[]
 
-
-You will need to save your completed file with the file name *_metrics.yaml_*.
+Copy this file to another directory with the file name
+*_metrics-deployer.yaml_*.
```

Comment:
Updated for clarity to: "In case you need to make any changes to this file later, copy it to another directory with the file name metrics-deployer.yaml and refer to the new location when using it in the following sections."